### DISCUSSION TITLE
Trying to use @stylistic/padded-blocks - is there a way to debug when it doesn't work?

### DISCUSSION THREAD

**Discussion by matthew-dean** [eslint-stylistic/eslint-stylistic#623] [Q&A]
I'm migrating from older ESLint configs and applying Stylistic from older style rules.

I'm applying linting to the config file itself and I noticed that it doesn't remove padded blocks from this:
```
{
    ...love,
    extends: configs.recommended,
    plugins: {
      '@stylistic': stylistic
    },
    files: ['**/*.{js,cjs,mjs}'],
    languageOptions: {
      ecmaVersion: 'latest',
      sourceType: 'module',
      globals: {
        ...globals.node,
        ...globals.browser
      }
    },
    settings: {
      'import/resolver': {
        typescript: {}
      },

      'import/extensions': ['.js', '.ts']
    },
    rules: {
      ...stylisticRules,
      '@stylistic/operator-linebreak': ['error', 'before', {
        overrides: {
          '=': 'after'
        }
      }],
      '@stylistic/function-call-spacing': 'error',
      '@stylistic/space-before-function-paren': ['error', {
        anonymous: 'never',
        named: 'never',
        asyncArrow: 'always'
      }],
      camelcase: ['error', {
        properties: 'never',
        ignoreDestructuring: true,
        ignoreImports: true
      }],

      'import/no-absolute-path': 0,
      'import/no-unresolved': 'error',

      'max-len': ['warn', {
        code: 120,
        tabWidth: 2,
        ignoreComments: true,
        ignoreTrailingComments: true,
        ignoreStrings: true,
        ignoreTemplateLiterals: true,
        ignoreRegExpLiterals: true
      }],

      'no-return-assign': 0,

      'no-void': 0

    }
```

(Note the last line of the object.) I ran `npx eslint --print-config file.js > test.json` to confirm that, yes, the style rule was turned on, but wasn't being applied. Is there another way to debug this.

**Comment by 9romise** [comment]
Have you tried [@eslint/config-inspector](https://github.com/eslint/config-inspector)?

**Reply by matthew-dean** [reply]
Whoa, that's an extremely useful tool! However, I think it surfaced an ESLint bug? It says `eslint.config.mjs` was ignored by the glob (in `ignores`) "!.storybook". This is non-sensical and contrary to the documentation, which says that this should just be an "unignore". But, regardless, I removed it.

However, even though it shows the Stylistic rule as applying, it still doesn't remove that blank line. 🤷‍♂️  

That said, I went back and read the [rule](https://eslint.style/rules/js/padded-blocks) for this, and, while I was interpreting this multi-line object as a "block", I noticed that it's not explicitly called out that way. I went through other various spacing rules and I actually think there isn't a Stylistic rule for this?
